lib/option.js

```diff
@@ -12,12 +12,17 @@ class Option {
    * @api public
    */
 
-  constructor(flags, description, autocomplete) {
+  constructor(flags, description, options = {}) {
+    options = _.isPlainObject(options) ? options : {
```

Comment:
I'm trying to get lodash out of the project eventually - is it possible to use vanilla JS for this?